microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,97 @@
+
+## Avoiding potential misuse  
+We believe that **Limited** is the right option for most users. **Limited** respects the user's regional preferences while introducing the least amount of risk of the website misusing this information.
+
+#### Fingerprinting Entropy
+OS regional information could be misused to compromise the user's privacy.  The regional information would help hackers establish fingerprinting entropy. In combination with many other pre-existing bits of entropy in the platform today, the user could be uniquely identified leading to more potential secondary harm. While a user's region and language are already available via the web platform, users generally fall into well-recognized regional codes that are similar across geographies.  Because the sample sizes are so large, the risk of any specific regional code standing out is small, and thus the risk is small. If the user changes their OS preferences to a unique language + region, websites may be able to identify the anomaly in a population, and easily uniquely identify the associated user. 
+
+#### Content Presentation
+The OS regional information can also be misused and lead to inconsistency of a Website's language presentation. Microsoft Edge provides multiple language and regional APIs to Websites developers.  Websites may process regional information in one place, while deriving language preferences from another.
+
+When the language and regional information is self-consistent, then it doesn't really matter which APIs the website uses to assemble its content presentation. However, if a Website reads the language information from multiple APIs and those APIs return different language codes, then the website may inadvertently start to mix-and-match regional preferences, resulting in strange user experiences. Watch for phrasing that mixes languages and regional preferences together, such as "Email sent on miércoles, 5 de enero de 2021".
+
+<!-- ====================================================================== -->
+## Developer Details 
+The new setting to **Share additional OS regional format** directly impacts the JavaScript `Intl` object's default regional value and all other JavaScript APIs that use the default region (such as the `Date` constructor). 
+
+When the `Intl` object's constructors (such as `DateTimeFormat`) are provided with an existing regional code, their behavior is unchanged. However, when a specific regional code isn't provided, the constructors will use the **default regional code**. The new user preference also affects other date and time formatting APIs that rely on the default regional code, such as `Date`.
+
+The default regional code is what is surfaced via the new user preference. For example, a `DateTimeFormat` can be created with the user's default regional code (and inspected) via the following expression:
```

Comment:
```suggestion
For example, a `DateTimeFormat` can be created with the user's default regional code (and inspected) via the following expression:
```
Dropping the part that is redundant with the message in the last two paragraphs.